nixos/modules/services/cluster/k3s/default.nix

```
@@ -119,6 +120,9 @@ let
     ${builtins.concatStringsSep "\n" (map linkManifestEntry enabledManifests)}
     ${builtins.concatStringsSep "\n" (lib.mapAttrsToList linkChartEntry cfg.charts)}
     ${builtins.concatStringsSep "\n" (map linkImageEntry cfg.images)}
+
+    mkdir -p $(dirname ${containerdConfigTemplateFile})
+    ${pkgs.coreutils-full}/bin/ln -sfn ${pkgs.writeText "config.toml.tmpl" cfg.containerdConfigTemplate} ${containerdConfigTemplateFile}
```

```suggestion
    ${coreutils}/bin/ln -sfn ${pkgs.writeText "config.toml.tmpl" cfg.containerdConfigTemplate} ${containerdConfigTemplateFile}
```
Why `full`? It's a larger package: 42MB vs 59MB

---

Just mimicking the rest of this file, which uses `coreutils-full` throughout. Probably better to keep it consistent?

---

Then I wonder why the rest of the file uses `coreutils-full`. Yes, let's use a single way. But likely it should have been `coreutils` everywhere, unless there was a reason for that. (Which I don't know atm.)